src/util/apiPrefix.ts

```diff
@@ -1,5 +1,5 @@
 import isProd from "./isProd";
 
-const apiPrefix = isProd ? "https://cellxgene.cziscience.com" : "http://127.0.0.1:5000/api";
+const apiPrefix = isProd ? "https://cellxgene.cziscience.com" : "/api";
```

Comment:
in our meeting, I can explain how this works a little better, but since we are using path-based routing, we don't need to know the domain anymore